game/src/components/health.rs

```diff
@@ -0,0 +1,31 @@
+use bevy::prelude::*;
+
+///pointer struct to mark entity as alive
+#[derive(Component, Debug)]
+pub struct Alive;
+
+///pointer struct to mark entity as dead
+#[derive(Component, Debug)]
+pub struct Dead;
+
+///Health component to track entity health
+#[derive(Component, Debug)]
+pub struct Health {
+    pub maximum_health: u32,
+    pub current_health: u32,
```

Comment:
Please remove health suffix and change fields to `maximum` and `current`.

Reply:
Done